app/templates/components/public/social-links.hbs

```diff
@@ -1,4 +1,7 @@
 <div class="ui list">
+  <div style="margin-bottom:6px;"><li>{{header-date @eventtime}} {{@eventtimezone}} To <span>{{header-date @eventtimeends}} {{@eventtimezone}}</span></li>
+  <a href={{this.calender}}>Add To Calender</a>
+  </div>
```

Comment:
This should be above social links, not in social links component itself

Reply:
so u want to me to create a new file add-to-calender.hbs. right? and should be  above social links

Reply:
Yes